### PR TITLE
examples: Update front-proxy image and add listener with upstream TLS context

### PR DESCRIPTION
**Pull request by dio**
Commit Message: This updates the front-proxy docker-compose illustration and adds a listener upstream TLS context configured.

Signed-off-by: Dhi Aurrahman <dio@tetrate.io>

Additional Description:

image

Risk Level: N/A
Testing: N/A
Docs Changes: This is a docs change.
Release Notes: N/A

### CODE REVIEW

File: docs/root/start/sandboxes/front_proxy.rst [hunk 1]
```diff
@@ -3,26 +3,25 @@
...
-service Envoys (routes setup in :repo:`/examples/front-proxy/front-envoy.yaml`). In turn the service
-envoys route the  request to the flask app via the loopback address (routes setup in
-:repo:`/examples/front-proxy/service-envoy.yaml`). This setup
-illustrates the advantage of running service Envoys  collocated with your services: all requests are
-handled by the service Envoy, and efficiently routed to your services.
+the edge of the ``envoymesh`` network. Port ``8080``, ``8443``, and ``8001`` are exposed by docker
+compose (see :repo:`/examples/front-proxy/docker-compose.yaml`) to handle ``HTTP``, ``HTTPS`` calls
+to the services and requests to ``/admin`` respectively. Moreover, notice that all traffic routed
+by the front Envoy to the service containers is actually routed to the service Envoys
+(routes setup in :repo:`/examples/front-proxy/front-envoy.yaml`). In turn the service envoys route
```
Comment (junr03) [inline docs/root/start/sandboxes/front_proxy.rst:21]: nit: capitalization of `Envoy` is inconsistent

**Comment by dio** [comment]
@junr03 when you have time, could you re-approve this? That was a failure re: FIPS and CI flakes. Thank you!

**Pull request merged**